components.css

```diff
@@ -61,4 +61,144 @@
 
 #info-alert a:hover{
     color: red;
+}
+
+.circular-x-large-avatar{
+    box-shadow: 5px 5px 15px var(--black);
+    margin: 1rem;
+    height: 8rem;
+    border-radius: 50%;
+}
+
+.circular-large-avatar{
+    box-shadow: 5px 5px 15px var(--black);
+    height: 6rem;
+    border-radius: 50%;
+    margin: 1rem;
+}
+
+.circular-small-avatar{
+    box-shadow: 5px 5px 15px var(--black);
+    margin: 1rem;
+    height: 4rem;
+    border-radius: 50%;
+}
+
+.circular-x-small-avatar{
+    box-shadow: 5px 5px 15px var(--black);
+    margin: 1rem;
+    height: 3rem;
+    border-radius: 50%;
+}
+
+.square-x-large-avatar{
+    box-shadow: 5px 5px 15px var(--black);
+    margin: 1rem;
+    height: 8rem;
+    border-radius: 1rem;
+}
+
+.square-large-avatar{
+    box-shadow: 5px 5px 15px var(--black);
+    height: 6rem;
+    border-radius: 1rem;
+    margin: 1rem;
+}
+
+.square-small-avatar{
+    box-shadow: 5px 5px 15px var(--black);
+    margin: 1rem;
+    height: 4rem;
+    border-radius: 1rem;
+}
+
+.square-x-small-avatar{
+    box-shadow: 5px 5px 15px var(--black);
+    margin: 1rem;
+    height: 3rem;
+    border-radius: 1rem;
```

Comment:
you can use utility classes for border-radius, as you are applying same proprty again and again in every declaration block.

Reply:
ok will follow that, thanks for the review.